src/common/kits/GlobalManager.ts

```
@@ -33,26 +33,57 @@ class GlobalManager {
+            dev: "https://auth.ginkgoo.dev",
+            staging: "https://staging-auth.xeni.legal",
+            prod: "",
+            local: "https://auth.ginkgoo.dev",
+            // local: "https://staging-auth.xeni.legal",
+          }[this.instance.g_webenv] || "",
+        apiServerUrl:
+          {
+            dev: "https://api.ginkgoo.dev/api",
+            staging: "https://staging-api.xeni.legal/api",
+            prod: "",
+            local: true ? "https://api.ginkgoo.dev/api" : "http://localhost:8080/api",
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)